apps/web/src/server/applications/case/documentation-metadata/__tests__/documentation-metadata.test.js

```diff
@@ -401,13 +401,13 @@ describe('Edit applications documentation metadata', () => {
 				const element = parseHtml(response.text);
 
 				expect(element.innerHTML).toMatchSnapshot();
-				expect(element.innerHTML).toContain('Enter the published date');
+				expect(element.innerHTML).toContain('You must enter the published date');
 			});
 
 			it('should return an error if the day is not valid', async () => {
 				const response = await request.post(`${baseUrl}/published-date`).send({
 					'datePublished.day': '99',
-					'datePublished.month': '1',
+					'datePublished.month': '01',
```

Comment:
I had to apply the new constraint (DD/MM/YYYY) to every other page with a date input: documentation metadata and key dates in the 'create new case' journey.